### ISSUE TITLE
Feature: Add hotkey to choose image browser

### ISSUE THREAD

**Issue by Nedrapter** [SvenWerlen/moulinette-core#56]
When clicking on an actor's image, sometimes you simply want to change the path (cause you have one on your clipboard) and sometimes you want to browse your Moulinette Library. Alternating which browser type will open by holding down a hotkey (e.g. Shift+mouse click) would help a lot

**Comment by SvenWerlen** [comment]
I don't think that I can implement this because FVTT implements the call (for opening the FilePicker). 
Do you know that you can switch back to default (there is a button for it)?
![image](https://user-images.githubusercontent.com/938060/215931065-bafb90a0-f22c-42bd-85a1-03123a0fe77e.png)

I understand that you want to save that extra step

**Comment by SvenWerlen** [comment]
I actually found a way to implement this. It will be part of version `moulinette-core` `10.6.0` which should be released on February 5.

**Comment by Nedrapter** [comment]
Yea I knew about the icon. The reasoning behind my request is that sometimes I have a url in my clipboard ready to paste as token image, but when I click the portrait, I have to wait for moulinette to load, open, then click on the icon and paste the image. And being hosted on the Forge, the load isn't very fast.
